aws/aws-how-to/instances/create-ec2-using-ubuntu-image.rst

```diff
@@ -0,0 +1,112 @@
+
+Please follow `Create key pairs <https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/create-key-pairs.html>`_ to create key pair and `Configuration and credential file settings <https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html>`_ for AWS CLI.
+
+Official Ubuntu Cloud Guest Amazon Machine Images (AMIs)
+--------------------------------------------------------
+
+The Official AMI Ids can be obtained from `Find Ubuntu images on AWS <https://canonical-aws.readthedocs-hosted.com/en/latest/aws-how-to/instances/find-ubuntu-images/>`_. Official Ubuntu AMIs are published by the 'Canonical' user, with Amazon ID '099720109477'. Images containing the string 'ubuntu' but not owned by that ID are not official AMIs.
+
+EC2 security groups
+-------------------
+
+Security groups allow you to specify firewalling rules for your instances. These firewalling rules are independent of, and in addition to, the software firewalling provided by the instance's operating system (iptables in the case of modern Ubuntu systems). Security groups must be defined before you create the instances that you would like to be members of those security groups. You specify the security groups to add an instance to at creation time with the -security-group-ids or security-groups option to the `run-instances <https://docs.aws.amazon.com/cli/latest/reference/ec2/run-instances.html>`_ command. You cannot add an existing instance to a security group.
+
+How you set up your security groups is up to you. You may choose to set up security groups that correspond to server functions, or have a separate security group for each instance. An instance may be a member of multiple security groups. If you don't specify any security groups when you instantiate an instance, it will be added to the default security group. Our examples use the default security group, but keep in mind that this means that this causes an inability to set up firewalling rules in a granular fashion.
+
+To create, delete and describe your security-group, you can use command below.
+
+.. code::
+
+    aws ec2 create-security-group --group-name <group name> --description <description>
+    aws ec2 delete-security-group --group-id <group ids>
+    aws ec2 describe-security-groups --group-ids <group-ids>
+
+See the `Using Security Groups <https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-security-groups.html>`_ of Using Security Groups section of the `User Guide for Amazon EC2 <https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/concepts.html>`_ for more information.
+
+Instantiating an image
+----------------------
+
+To start an instance, we use a command of the following form:
```

Comment:
Change to something like: 
Using the key pair and AMI ID obtained above, launch an instance by running: